packages/tools/pie-wrapper-react/custom-elements.json

```diff
@@ -326,6 +357,15 @@
           },
           "default": "['h1', 'h2', 'h3', 'h4', 'h5', 'h6']",
           "description": "Modal heading levels/tags"
+        },
+        {
+          "kind": "variable",
+          "name": "ON_MODAL_CLOSE_EVENT",
```

Comment:
I think I missed when this was all added, but it seems... suboptimal that we need to update this package any time something changes in any component.

Reply:
I believe this was auto added.

Reply:
For context – this gets added as part of the react-wrapper so that reactive properties get mapped as part of the react component build.

This is autogenerated by the custom elements package that runs as part of the react-wrapper build step